### PR TITLE
-updated kick-out page

### PR DESCRIPTION
**Pull request by hemerem-ch**
-update content for you cannot use this service

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: locales/cy/stop-not-relevant-officer.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
     "cannotUseServiceTitle" : "You cannot use this service welsh",
-    "listOfAuthorisedAgent" : "To register an authorised agent, you must be one of the following: welsh",
+    "listOfAuthorisedAgent" : "To register an authorised agent, you must be one of the following for the business you are registering:welsh",
     "kickoutDirector": "director welsh",
     "kickoutGeneralPartner": "general partner welsh",
-    "kickoutPartner": "partner welsh",
+    "kickoutPartner": "member welsh",
```
Comment (swalke-ch) [inline locales/cy/stop-not-relevant-officer.json:6]: Can we update the keys as well just to avoid confusion ?

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by swalke-ch — DISMISSED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by sampankumar — APPROVED** [review]

**Review by abhiahirv1 — APPROVED** [review]

**Pull request merged**